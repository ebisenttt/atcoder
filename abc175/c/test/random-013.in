-725202325293516 961651984360350 604758963273265
