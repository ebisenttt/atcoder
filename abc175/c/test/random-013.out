484315601253014
